app/assets/stylesheets/components/_foreword.scss

```diff
@@ -1,31 +1,28 @@
-$gutter: 40px;
 $trigger-width: 30px;
 $trigger-height: $trigger-width;
 
 .foreword {
-  padding: $gutter-large $gutter-small;
+  padding: 32px $gutter-small;
```

Comment:
So looking at the use of 32 px here, I can see that I made these changes in response to a comment @WKocur made (I think) about only using 'gutter' variables for horizontal spacing (at the time `$gutter-large` was being used where a lot of these instances of `32px` are now. At the end of that conversation I decided that there wasn't enough commonality between all these instances to justify a variable, but I'm not over the moon about the repetition either...

[Here's a link to the conversation](https://github.com/unepwcmc/wcmc-annual-review/pull/42/#discussion_r646498543). I think because it was only 4 instances of 32px, that also led me to think it wasn't worth its own variable.

At this point, on balance, I would be in favour of letting this one slide. Could easily keep that 32px in a variable, which would make the code more DRY, but at the same time it's not necessarily true that we want all these values to be tied together since they're unrelated.

What do you think?

Reply:
Makes sense -- if they're unrelated and it's just a coincidence then makes sense. I suppose the issue is then that they're not coming from any kind of centralised defaults so it highlights a different issue, but might not be worth your time